darwin/clean.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -xe
+
+cd "$WORKSPACE_PATH/../../.." || exit 1
+echo "Executing gmake clean in $(pwd)..."
+export PATH="$(brew --prefix)"/bin:"$PATH"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)